AsyncSwift/Views/EventView.swift

```diff
@@ -5,16 +5,153 @@
 //  Created by Kim Insub on 2022/09/06.
 //
 
+/* TODO: 내용
+ 1. light mode only
+ 2. lock landscape
+ 2. 리스트의 세버론 웨이트 라이트로 적용하기
+ 3. 디바이더 너비 키우기
+ 4. 예정된 행사 테그 구현하기
+ */
+
 import SwiftUI
 
+struct EventModel {
+
+    var event: Event
+    var sessions: [Session]
+
+    struct Event {
+
+        var title: String
+        var detailTitle: String
+        var subject: String
+        var description: [Paragraph]
+        var date: String
+        var startDate: String
+        var endDate: String
+        var time: String
+        var location: String
+        var address: String
+        var hashTags: String
+        var addressURLs: AddressURLs
+
+        struct Paragraph: Hashable {
+            var content: String
+        }
+
+        struct AddressURLs {
+            var naverMapURL: String
+            var kakaoMapURL: String
+        }
+    }
+
+    struct Session: Identifiable {
+
+        let id: Int
+        var title: String
+        var description: [Paragraph]
+        var speaker: Speaker
+
+        struct Speaker {
+            var name: String
+            var imageURL: String
+            var role: String
+            var description: String
+        }
+
+        struct Paragraph: Hashable {
+            var content: String
+        }
+    }
+}
+
+
 struct EventView: View {
+
+    private let data = Mock.data
+
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        NavigationView {
+            ScrollView {
+                Header
+                LazyVStack {
+                    ForEach(data.sessions) { session in
+                        makeSessionCell(for: session)
+                    }
+                }
+            }
+            .navigationTitle(Tab.event.title)
+        }
     }
 }
 
-struct ScheduleView_Previews: PreviewProvider {
-    static var previews: some View {
-        EventView()
+private extension EventView {
+
+    var Header: some View {
+        VStack(alignment: .leading, spacing: 8) {
+            Text(data.event.subject)
+                .font(.title)
+                .fontWeight(.bold)
+            HStack {
+                Text(data.event.title)
+                    .font(.caption2)
+                    .fontWeight(.bold)
```

Comment:
```suggestion
                    .font(.system(size:11, weight: .bold))
```
font weight가 중복되어서 선언됩니다!

Reply:
이또한 아래 댓글들과 동일한 내용입니다